src/main/java/salesforce/lightning/WorkTypeForm.java

```diff
@@ -0,0 +1,71 @@
+package salesforce.lightning;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import salesforce.BasePage;
+
+public class WorkTypeForm extends BasePage {
+
+    @FindBy(css = ".select[aria-required='true']")
+    protected WebElement estimatedDurationComboBox;
+
+    @FindBy(xpath = "//button[@data-aura-class='uiButton forceActionButton'][3]")
+    protected WebElement saveBtn;
+    private String xpathComboBox;
+    private static int countComboBox = 0;
+
+    public WorkTypeForm(final WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    protected void waitForPageLoaded() {
+        webElementAction.waitForElementVisibility(estimatedDurationComboBox);
+    }
+
+    public void setInputField(final String fieldName, final String fieldValue) {
+        if ("Description".equals(fieldName)) {
+            webElementAction.setInputField(driver.findElement(By.cssSelector(".textarea")), fieldValue);
+        } else {
+            String xpath = String.format("//*[contains(text(),'%s')]/../..//*[@type='text']", fieldName);
```

Comment:
Since the Path string will not change, maybe you could set it as a final var in the class, so you could re use it